test/e2e/specs/launch.spec.ts

```diff
@@ -1,29 +1,29 @@
 import fs from 'fs';
 import path from 'path';
 import { testWithSpectron } from 'vue-cli-plugin-electron-builder';
+import spectron from 'spectron';
 import {
   Application, SpectronClient, SpectronWindow, StopServe
 } from '../type';
-import Interactions from '../utils/interactions';
+// import Interactions from '../utils/interactions';
 
 const screenshotsDir = './screenshots';
 
-jest.setTimeout(100000);
```

Comment:
Why do we need this again? Worth writing a comment

Reply:
I don't know why we need to increase, it something to investigate.
But I very want to keep the test, even if it just tests the "title", because it tests the e2e.